src/components/chat/MessageBox.js

```diff
@@ -16,8 +15,14 @@ class MessageBox extends Component {
     };
   }
 
+  onNewMessages = () => {
+    if (this.props.isAudioEnabled){
+      playNotification();
+    }
+    this.scrollToBottom();
```

Comment:
Ancillary note: if I'm reading this correctly, I could see this becoming a nuisance if a user is scrolling back up to read past messages and they keep getting pulled back down.

Reply:
Yup, I've experienced that. We could add a "Scroll to bottom" floating overlay button in the bottom right when the user scrolls up, and not automatically scroll down when a new message comes in...